analysis-engine/src/main/java/org/sonarsource/sonarlint/core/analysis/container/analysis/sensor/SensorsExecutor.java

```diff
@@ -47,18 +47,18 @@ public class SensorsExecutor {
 
   private final SensorOptimizer sensorOptimizer;
   private final ProgressMonitor progress;
-  private final List<Sensor> sensors;
+  private final List<ProjectSensor> sensors;
   private final DefaultSensorContext context;
 
-  public SensorsExecutor(DefaultSensorContext context, SensorOptimizer sensorOptimizer, ProgressMonitor progress, Optional<List<Sensor>> sensors) {
+  public SensorsExecutor(DefaultSensorContext context, SensorOptimizer sensorOptimizer, ProgressMonitor progress, Optional<List<ProjectSensor>> sensors) {
     this.context = context;
     this.sensors = sensors.orElse(List.of());
     this.sensorOptimizer = sensorOptimizer;
     this.progress = progress;
   }
 
   public void execute() {
-    for (Sensor sensor : sort(sensors)) {
+    for (var sensor : sort(sensors)) {
```

Comment:
Isn't there a specific order to run `ProjectSensor`s that are not `Sensor`s? I thought since they are executed once, they should be executed after the other sensors.